socorro/scripts/sqs_cli.py

```diff
@@ -101,8 +102,19 @@ def publish(ctx, queue, message):
         click.echo("Queue %s does not exist.")
         return
 
-    conn.send_message(QueueUrl=queue_url, MessageBody=message)
-    click.echo("Message sent.")
+    # Pull crash ids from stdin if there are any
+    if not crashids and not sys.stdin.isatty():
+        crashids = list(click.get_text_stream("stdin").readlines())
+
+    if not crashids:
+        raise click.BadParameter(
+            "No crashids provided.", ctx=ctx, param="crashids", param_hint="crashids"
+        )
+
+    for crashid in crashids:
+        crashid = crashid.rstrip()
```

Comment:
We run `rstrip` here because if it's coming from stdin, then it'll end in carriage returns which we want to remove.